elements/drawtools/src/methods/controller/update-button-states.js

```diff
@@ -0,0 +1,28 @@
+import { nothing } from "lit";
+
+/**
+ * Update button state of draw and discard buttons based on specific conditions.
+ *
+ * @param {import("../../components/controller").EOxDrawToolsController} EoxDrawToolController - The controller instance.
+ * @returns {{ drawDisabled: boolean | import("lit").nothing, discardDisabled: boolean | import("lit").nothing }} - The updated button states.
+ */
+const updateButtonStatesMethod = (EoxDrawToolController) => {
+  // Destructure relevant properties from the controller
+  const { multipleFeatures, drawnFeatures, currentlyDrawing } =
+    EoxDrawToolController;
+
+  // Determine draw button's disabled state based on conditions
+  const drawDisabled =
+    (!multipleFeatures && drawnFeatures?.length > 0) ||
+    currentlyDrawing ||
+    nothing;
```

Comment:
Is there a concrete reason for returning a lit `nothing` here (and L22)? This adds a dependency to this otherwise vanilla JS module.
Could we use something like `?disabled="${this.#discardDisabled || nothing}"` in the controller.js instead?

Reply:
Yeah you are right, I shouldn't have imported `nothing` inside a method file which is contradicting. `nothing` should only be part of components because lit it associated with that.

I will add it to `controller.js`

Reply:
Done ✅